### ISSUE TITLE
 could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR when using gpu

### ISSUE THREAD

**Issue by mingyangShang** [smallcorgi/Faster-RCNN_TF#124]
My environment is:
Ubuntu 16.04 
Tensorflow-gpu 0.12
Cuda :8.0
Cudnn: 5.1.10

When run the demo in tools/demo.py, such error occurs:
`E tensorflow/stream_executor/cuda/cuda_dnn.cc:397] could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR
E tensorflow/stream_executor/cuda/cuda_dnn.cc:364] could not destroy cudnn handle: CUDNN_STATUS_BAD_PARAM
F tensorflow/core/kernels/conv_ops.cc:605] Check failed: stream->parent()->GetConvolveAlgorithms(&algorithms) `
I have googled about this, but didn't find a solution, Has this error occurred to others?

**Comment by 714586886** [comment]
I met a similar error:failed to create cublas handle.
Env:
Ubuntu 16.04
Tensorflow-gpu 1.0.1
CUDA 8.0
Cudnn 5.1
I think the error is caused by CUDA or the version of tensorflow.

**Comment by mingyangShang** [comment]
@714586886, yeah, i have the same environment with yours in my server machine, the same error occurs. Have you solved this problem?

**Comment by 714586886** [comment]
No solution...I have tried to solve it by changing Tensorflow version or re-install CUDA, but it does not work.

**Comment by SaverioSun** [comment]
I have met the same question,can anybody help me

**Comment by SunTiecheng** [comment]
me too

**Comment by coderzbx** [comment]
same there's no solution

**Comment by adityakpranav** [comment]
I got the same error in windows 10 console but same code ran on Spyder IDE without any error.

**Comment by MichalDulemba** [comment]
Anyone solved this issue?

**Comment by liuxinhai** [comment]
I got the same issue. Anyone can help?

**Comment by liuxinhai** [comment]
I figured out this issue by using a larger GPU memory. I used more than 8Gbs on GPU.
